css/style.css

```diff
@@ -1,19 +1,100 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* header (nav) */
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+.nav_bar {
+  background-color: white;
+  position: fixed;
+  width: 100%;
+  height: 4.5rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  top: 0;
+}
+
+.nav_logo {
+  height: 2.2em;
+  padding-top: 0.5em;
+  margin-left: 7em;
+}
+
+.nav_list {
+  font-size: 1rem;
+}
+
+.nav_list li {
+  display: inline-block;
+  margin-left: 1rem;
+  color: grey;
+}
+
+.nav_list .highlighted {
+  font-weight: 500;
+  color: rgb(75, 73, 73);
+}
+/* intro main */
+
+.intro {
+  background-image: url("../img/first-background.jpg");
+  background-position: center;
+  height: 45rem;
```

Comment:
Depending on which device you're using, an absolute value of 45rem might not always be appropriate. You could use a percentage or viewport units instead.